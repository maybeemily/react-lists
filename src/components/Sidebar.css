.Sidebar {
  display: flex;
  flex-direction: column;
  width: 20vw;
  height: 100vh;

  background-color: olivedrab;
  color: white;
  text-align: center;

  a {
    color: white;
    font-size: 1.5rem;
  }
}
