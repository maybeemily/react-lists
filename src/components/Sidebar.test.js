import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar component', () => {
  it('renders a sidebar', () => {
    const wrapper = shallow(<Sidebar><p>test paragraph</p></Sidebar>);
    expect(wrapper).toMatchSnapshot();
  });
});
