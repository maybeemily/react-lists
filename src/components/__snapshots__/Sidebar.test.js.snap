// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component renders a sidebar 1`] = `
<nav
  className="Sidebar"
>
  <p>
    test paragraph
  </p>
</nav>
`;
